drivers/segger/sysview.c

```diff
@@ -22,55 +22,94 @@
  * Included Files
  ****************************************************************************/
 
-#include <nuttx/config.h>
-#include <syslog.h>
-
 #include <nuttx/clock.h>
 #include <nuttx/sched.h>
 #include <nuttx/sched_note.h>
+#include <nuttx/note/note_driver.h>
 #include <nuttx/segger/sysview.h>
 
+#include <stddef.h>
+#include <syslog.h>
 #include <SEGGER_RTT.h>
 #include <SEGGER_SYSVIEW.h>
 
 #include "sched/sched.h"
 
 /****************************************************************************
- * Private Types
+ * Private Functions
  ****************************************************************************/
 
-struct sysview_s
-{
-  unsigned int                 irq[CONFIG_SMP_NCPUS];
-#ifdef CONFIG_SCHED_INSTRUMENTATION_FILTER
-  struct note_filter_mode_s    mode;
+static void sysview_start(FAR struct note_driver_s *drv,
```

Comment:
let's add note_ prefix to all struct, function, variable and file name(note_sysview.c)

Reply:
done